woocommerce/woocommerce
# [COT] Implement order preview functionality and edit by clicking order row

### All Submissions:






### Changes proposed in this Pull Request:



Closes #34596.

### How to test the changes in this Pull Request:

1. Make sure COT is disabled.
2. Go to WC > Orders and make sure that the following are working correctly:
   1. Clicking anywhere on an order row should take you to the order edit page.
   2. Clicking on the preview icon should trigger a modal window with order details and buttons to change the order status and edit. All these buttons should be working.

      https://user-images.githubusercontent.com/184724/191596294-615e12fb-3e8e-48da-9ae3-7e8561847d62.mov
3. Enable COT.
4. Repeat step 2 but notice that all the links/buttons should now take you to the corresponding COT pages instead of post-based ones.

### Other information:




### FOR PR REVIEWER ONLY:
